plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/shell/ShellCommand.java

```diff
@@ -180,6 +181,41 @@ private static Thread copyStream(InputStream inputStream, OutputStream outputStr
         }
         return null;
     }
+    
+    private static class UpdateObserverRunnable implements Runnable {
```

Comment:
This logic looks copy/pasted from above.  Any way to avoid the duplication?

Reply:
This change is actually not needed and not used since separate thread was created in Observer. Sorry I forgot to clean this up. Will remove.